incubator/hnc/internal/pkg/selector/selector.go

```diff
@@ -0,0 +1,67 @@
+package selector
```

Comment:
A package-level comment is a good practice.

```
// Package selector is ...
```

You start with the words "Package \<packagename\>" for Godoc: https://blog.golang.org/godoc

Reply:
done